sdk/tables/Azure.Data.Tables/api/Azure.Data.Tables.netstandard2.0.cs

```diff
@@ -29,14 +29,19 @@ public enum ServiceVersion
     public partial class TableServiceClient
     {
         protected TableServiceClient() { }
+        public TableServiceClient(System.Uri endpoint, Azure.Data.Tables.TableClientOptions options = null) { }
         public TableServiceClient(System.Uri endpoint, Azure.Data.Tables.TablesSharedKeyCredential credential, Azure.Data.Tables.TableClientOptions options = null) { }
         public virtual Azure.Data.Tables.Models.TableResponse CreateTable(string tableName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Data.Tables.Models.TableResponse> CreateTableAsync(string tableName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual Azure.Response DeleteTable(string tableName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual System.Threading.Tasks.Task<Azure.Response> DeleteTableAsync(string tableName, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response<System.Collections.Generic.IReadOnlyList<Azure.Data.Tables.Models.SignedIdentifier>> GetAccessPolicy(string table, int? timeout = default(int?), string requestId = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual System.Threading.Tasks.Task<Azure.Response<System.Collections.Generic.IReadOnlyList<Azure.Data.Tables.Models.SignedIdentifier>>> GetAccessPolicyAsync(string table, int? timeout = default(int?), string requestId = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual Azure.Data.Tables.TableClient GetTableClient(string tableName) { throw null; }
         public virtual Azure.Pageable<Azure.Data.Tables.Models.TableResponseProperties> GetTables(string select = null, string filter = null, int? top = default(int?), System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
         public virtual Azure.AsyncPageable<Azure.Data.Tables.Models.TableResponseProperties> GetTablesAsync(string select = null, string filter = null, int? top = default(int?), System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
+        public virtual Azure.Response SetAccessPolicy(string table, int? timeout = default(int?), string requestId = null, System.Collections.Generic.IEnumerable<Azure.Data.Tables.Models.SignedIdentifier> tableAcl = null, System.Threading.CancellationToken cancellationToken = default(System.Threading.CancellationToken)) { throw null; }
```

Comment:
Does `requestId` correspond to `x-ms-client-request-id`?  If so, not sure we want to expose it quite like this.  See https://github.com/Azure/azure-sdk-for-net/issues/11273 that's tracking the discussion.

Reply:
And `timeout` is a thing we don't expose from .NET.  The correct way to handle that is via your `cancellationToken`.  It would be nice to pass along to the service, but there's no way to read the remaining time on a CancellationToken.

Reply:
timeout is an odd optional argument supported by some table APIs [that interpreted by the service. ](https://docs.microsoft.com/en-us/rest/api/storageservices/setting-timeouts-for-table-service-operations). It has some specific restrictions such as defaulting to 30 seconds, and ignoring values greater that 30 seconds.  

Would you suggest wrapping this into the CancellationToken regardless?

Reply:
Yep, that's how it works for other Storage services too.  If you ever really need to change a server-side timeout, you can do it with a custom pipeline policy but that was obviously an advanced scenario.  Might be worth considering timeout to be another "optional advanced parameter" like clientRequestId and come up with a unified way of exposing it while not confusing customers by putting it on simpler APIs.

Reply:
tracking this with #11774